src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -195,7 +272,10 @@ class PaymentInformation extends React.Component {
           render={handleDowntimeForSection('payment information')}
           dependencies={[externalServices.evss]}
         >
-          <AdditionalInfos />
+          <AdditionalInfos
+            gaProfileAction="view-link"
```

Comment:
Since this component is only used once, do you need this to be a prop?

Reply:
Fixed.